lib/core/src/server/preview/entries.js

```diff
@@ -1,5 +1,34 @@
 import path from 'path';
 import { logger } from '@storybook/node-logger';
+import stable from 'stable';
+import { loadPreviewOrConfigFile } from '../utils/load-preview-or-config-file';
+
+export const sortEntries = entries => {
+  const isGenerated = /generated-(config|other)-entry/;
+  const isGeneratedConfig = /(?:preview|config)\..+-generated-config-entry/;
+
+  return stable(entries.slice(0), (a, b) => {
```

Comment:
"preview & config can call configure which should always happen AFTER all global decorators & args are set." - better explanation